presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -95,6 +95,7 @@
     private DataSize s3MultipartMinPartSize = new DataSize(5, MEGABYTE);
     private boolean useParquetColumnNames;
     private boolean pinS3ClientToCurrentRegion;
+    private String s3UserAgentPrefix = "";
```

Comment:
This is a bit weird to have an empty string as the default for a config property. Can we make this null by default?